server/docker.go

```diff
@@ -255,13 +369,24 @@ func (w *dockerWatcherImpl) parseContainerData(container *dockertypes.Container)
 		}
 		if key == DockerRouterLabelNetwork {
 			if data.network != nil {
-				logrus.WithFields(logrus.Fields{"containerId": container.ID, "containerNames": container.Names}).
+				logrus.WithFields(logrus.Fields{"containerId": container.ID, "containerNames": container.Name}).
 					Warnf("ignoring container with duplicate %s label", DockerRouterLabelNetwork)
 				return
 			}
 			data.network = new(string)
 			*data.network = value
 		}
+		if key == DockerRouterLabelAutoScaleUp {
+			lowerValue := strings.TrimSpace(strings.ToLower(value))
+			data.autoScaleUp = lowerValue != "" && lowerValue != "0" && lowerValue != "false" && lowerValue != "no"
```

Comment:
It would be great to create a helper function for this and then use the same at

https://github.com/itzg/mc-router/blob/22ec39b805bf32e473e21de70fb3b629ae5840dc/server/k8s.go#L287-L291

and below at L385